docs/content/components/caption.mdx

```diff
@@ -8,7 +8,9 @@ The caption component can be used to append a caption to images used in document
 
 ```jsx live
 <Flex alignItems="center" flexDirection="column">
-  <img src="https://s3.us-west-2.amazonaws.com/photos.puppyspot.com/breeds/245/card/500000291_medium.jpg" />
+  <img alt="" src="https://s3.us-west-2.amazonaws.com/photos.puppyspot.com/breeds/245/card/500000291_medium.jpg" />
   <Caption>A beautiful husky puppy.</Caption>
 </Flex>
 ```
+
+_Note: Be sure to include an ```alt``` attribute on your image. If the caption provides enough detail, ```alt=""``` should be used_
```

Comment:
Do we need three backticks for inline code?
```suggestion
_Note: Be sure to include an `alt` attribute on your image. If the caption provides enough detail, `alt=""` should be used_
```

Reply:
Nope, I'll clean it up here and in do/dont. Thanks!